PowerEditor/installer/APIs/nsis.xml

```diff
@@ -203,7 +236,6 @@
 		<KeyWord name="SetRegView" />
 		<KeyWord name="SetShellVarContext" />
 		<KeyWord name="SetSilent" />
-		<KeyWord name="SetStaticBkColor" />

```

Comment:
It seems a valid API too.

Reply:
Replaced in 2003: https://sourceforge.net/p/nsis/code/2347/